crates/next-api/src/nft_json.rs

```diff
@@ -104,15 +184,20 @@ impl Asset for NftJsonAsset {
 
         let output_root_ref = this.project.output_fs().root().await?;
         let project_root_ref = this.project.project_fs().root().await?;
+        let next_config = this.project.next_config().await?;
+
+        // Parse outputFileTracingIncludes and outputFileTracingExcludes from config
+        let output_file_tracing_includes = next_config.output_file_tracing_includes.clone();
+        let output_file_tracing_excludes = next_config.output_file_tracing_excludes.clone();
```

Comment:
These should be turbotask getter functions on `next_config()`, so that you don't await the entire `next_config()` and changing some unrelated value invalidates everything